src/main/webapp/main/App.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Routes from "./Routes";
+import LoadingWrapper from "../components/LoadingWrapper";
+
+const AppWithLoading = LoadingWrapper(Routes);
+
+export default function App() {
+
+    return (
+      AppWithLoading({
```

Comment:
Question ... do we need a constant for this? Can you just inline the LoadingWrapper in the return? Something like this:
```
return (LoadingWrapper(Routes));
```
I point this out because `AppWithLoading` is awkward. Tomorrow, if someone wants another app-level HOC, we then have something like 
```
const AppWithLoading = LoadingWrapper(Routes);
const AppWithMyNewFeature = MyNewFeature(AppWithLoading);
... etc.
```

Reply:
This way to handle it is now deprecated, no need for any constants.